plugin/src/kiali/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -591,6 +626,23 @@ export const SummaryPanelNode: React.FC<SummaryPanelNodeProps> = (props: Summary
     setIsKebabOpen(isOpen);
   };
 
+  const netObs = externalServices?.find(s => s.name && s.url && s.name.toLowerCase().includes('observ'));
+  const netObsBase = netObs?.url ? netObs.url.replace(/\/$/, '') : undefined;
+  
+  // Only construct netflow URL if Network Observability plugin is available and we have valid namespace data
+  let netObsUrl: string | undefined;
+  const namespace = nodeData.namespace;
+  
+  if (isNetobservAvailable() && namespace) {
+    if (netObsBase) {
+      netObsUrl = `${netObsBase}/netflow-traffic?timeRange=300&limit=5&match=all&showDup=false&packetLoss=all&recordType=flowLog&dataSource=auto&filters=src_namespace%3D${encodeURIComponent(namespace)}&bnf=false&function=last&type=Bytes`;
+    } else {
+      // For local development, construct URL relative to current host
+      const currentHost = window.location.origin;
+      netObsUrl = `${currentHost}/netflow-traffic?timeRange=300&limit=5&match=all&showDup=false&packetLoss=all&recordType=flowLog&dataSource=auto&filters=src_namespace%3D${encodeURIComponent(namespace)}&bnf=false&function=last&type=Bytes`;
```

Comment:
I think this is the only logic that ever executes, because netobserv is not defined as an external_service.